src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/FagsakDeltagerSøk.tsx

```diff
@@ -123,20 +120,20 @@ const FagsakDeltagerSøk: React.FC = () => {
                         : søkeresultat.harTilgang && settDeltagerForOpprettFagsak(søkeresultat)
                 }
             />
-
-            {innloggetSaksbehandler && (
-                <Endringslogg
-                    userId={innloggetSaksbehandler.navIdent}
-                    dataFetchingIntervalSeconds={60 * 15}
-                    appId={'BAKS'}
-                    backendUrl={'/endringslogg'}
-                    dataset={'production'}
-                    maxEntries={50}
-                    appName={'Barnetrygd'}
-                    alignLeft={true}
-                    stil={'lys'}
-                />
-            )}
+            {/*TODO: Legg inn Endringslogg når vi får konfigurert opp innhold for KS i familie-endringslogg (Sanity)*/}
+            {/*{innloggetSaksbehandler && (*/}
+            {/*    <Endringslogg*/}
+            {/*        userId={innloggetSaksbehandler.navIdent}*/}
+            {/*        dataFetchingIntervalSeconds={60 * 15}*/}
+            {/*        appId={'BAKS'}*/}
+            {/*        backendUrl={'/endringslogg'}*/}
+            {/*        dataset={'production'}*/}
+            {/*        maxEntries={50}*/}
+            {/*        appName={'Kontantstøtte'}*/}
+            {/*        alignLeft={true}*/}
+            {/*        stil={'lys'}*/}
+            {/*    />*/}
+            {/*)}*/}
```

Comment:
Trenger man én kommentar per linje?
```suggestion
            {/*TODO: Legg inn Endringslogg når vi får konfigurert opp innhold for KS i familie-endringslogg (Sanity) 
            {innloggetSaksbehandler && (
                <Endringslogg
                    userId={innloggetSaksbehandler.navIdent}
                    dataFetchingIntervalSeconds={60 * 15}
                    appId={'BAKS'}
                    backendUrl={'/endringslogg'}
                    dataset={'production'}
                    maxEntries={50}
                    appName={'Barnetrygd'}
                    alignLeft={true}
                    stil={'lys'}
                />
            )}*/
            }
```

Reply:
Syntes også det så rart ut 🙈 Vet ikke helt hvorfor det ble sånn 😛 Fikser opp i det!